subworkflows/local/utils_nfcore_metapep_pipeline/main.nf

```diff
@@ -0,0 +1,200 @@
+
+    //
+    // Print version and exit if required and dump pipeline parameters to JSON file
+    //
+    UTILS_NEXTFLOW_PIPELINE (
+        version,
+        true,
+        outdir,
+        workflow.profile.tokenize(',').intersect(['conda', 'mamba']).size() >= 1
+    )
+
+    //
+    // Validate parameters and generate parameter summary to stdout
+    //
+    pre_help_text = nfCoreLogo(monochrome_logs)
+    post_help_text = '\n' + workflowCitation() + '\n' + dashedLine(monochrome_logs)
+    def String workflow_command = "nextflow run ${workflow.manifest.name} -profile <docker/singularity/.../institute> --input samplesheet.csv --outdir <OUTDIR>"
+    UTILS_NFVALIDATION_PLUGIN (
+        help,
+        workflow_command,
+        pre_help_text,
+        post_help_text,
+        validate_params,
+        "nextflow_schema.json"
+    )
+
+    //
+    // Check config provided to the pipeline
+    //
+    UTILS_NFCORE_PIPELINE (
+        nextflow_cli_args
+    )
+    //
+    // Custom validation for pipeline parameters
+    //
+    validateInputParameters()
+
+    //
+    // Create channel from input file provided through params.input
+    //
+    ch_samplesheet = params.show_supported_models ? Channel.empty() : Channel.fromPath(input, checkIfExists: true)
+
+    emit:
+    samplesheet = ch_samplesheet
+    versions    = ch_versions
+}
+
+/*
+========================================================================================
+    SUBWORKFLOW FOR PIPELINE COMPLETION
+========================================================================================
+*/
+
+workflow PIPELINE_COMPLETION {
+
+    take:
+    email           //  string: email address
+    email_on_fail   //  string: email address sent on pipeline failure
+    plaintext_email // boolean: Send plain-text email instead of HTML
+    outdir          //    path: Path to output directory where results will be published
+    monochrome_logs // boolean: Disable ANSI colour codes in log output
+    hook_url        //  string: hook URL for notifications
+    multiqc_report  //  string: Path to MultiQC report
+
+    main:
+
+    summary_params = paramsSummaryMap(workflow, parameters_schema: "nextflow_schema.json")
+
+    //
+    // Completion email and summary
+    //
+    workflow.onComplete {
+        if (email || email_on_fail) {
+            completionEmail(summary_params, email, email_on_fail, plaintext_email, outdir, monochrome_logs, multiqc_report.toList())
+        }
+
+        completionSummary(monochrome_logs)
+
+        if (hook_url) {
+            imNotification(summary_params, hook_url)
+        }
+    }
+}
+
+/*
+========================================================================================
+    FUNCTIONS
+========================================================================================
+*/
+//
+// Check and validate pipeline parameters
+//
+def validateInputParameters() {
+    // Exit if running this pipeline with -profile conda / -profile mamba
```

Comment:
Why? :O

Reply:
It wasn't running before and I haven't tested it yet, so it's still on the list and will soon be deleted in a separate PR!